### PR TITLE
[MINOR] improve(doc): text changes to improve readability

### PR DESCRIPTION
**Pull request by cottage14**
Have removed duplications, changed the formatting in a couple of places to make it easier to understand options, and simplified the text to make it easier to read.

### CODE REVIEW

**Comment by cottage14** [comment]
I am the Editor-Writer for the Apache Software Foundation's Infrastructure team. Justin McLean asked if I could review the documentation for your excellent project.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This project includes the following roles.
 
 **3.2. Balance**. The development process should balance the interests of all stakeholders.
 
-**3.3. Harmonization**. Good faith efforts shall be made to resolve any conflicts.
+**3.3. Harmonization**. Good-faith efforts shall be made to resolve any conflicts.
 
 ## 4. Trademarks
```
Comment (justinmclean) [inline GOVERNANCE.md:31]: As this is based on someone else document I left it as it was, but I've no objections to that improvements.

File: docs/getting-started.md [hunk 2]
```diff
@@ -51,22 +55,22 @@ If you are using AWS and want to access the instance remotely, be sure to read [
    - Guidelines for accessing Gravitino externally when deployed on AWS, including necessary configurations and considerations.
 
 10. **Next steps**
-    - Concluding thoughts and suggesting next steps for users who have completed the setup.
+    - Concluding thoughts and suggested next steps for users who have completed the setup.
```
Comment (justinmclean) [inline docs/getting-started.md:58]: Probably remove "Concluding thoughts and" as there are no real concluding thoughts?

File: docs/getting-started.md [hunk 3]
```diff
@@ -225,27 +224,26 @@ newgrp docker
...
+1. Create a Metalake:
 
    ```shell
    curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
    -H "Content-Type: application/json" \
    -d '{"name":"metalake","comment":"Test metalake"}' http://localhost:8090/api/metalakes
    ```
 
-   Verify the MetaLakes creation
+   Verify the MetaLake's creation:
```
Comment (justinmclean) [inline docs/getting-started.md:246]: We probably should be consistent with Metalake vs MetaLake. I think we've mostly been using Metalake.

**Review by justinmclean — APPROVED** [review]
Changes all look good to me. Thanks for your contribution.

**Pull request merged**